recipes/somajo/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "somajo" %}
+{% set version = "2.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/SoMaJo-{{ version }}.tar.gz
+  sha256: 2d9cdf4fe191d75123a908038e84413407273150ce20242a2b1d1d2c1d1a49e9
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
```

Looks like this is pure Python and could be a noarch build:
```suggestion
build:
  script: {{ PYTHON }} -m pip install . -vv
  noarch: python
```